firehose.tf

```
@@ -0,0 +1,48 @@
+module "firehose" {
```

```suggestion
module "firehose_label" {
```

I think it's a better names since calling the module `firehose` is misleading